### DISCUSSION TITLE
Simulate workload with focus on specific wait_event_type?

### DISCUSSION THREAD

**Discussion by NikolayS** [lesovsky/noisia#7] [Ideas]
Great project, thank you

I wonder how difficult would it be to implement support for generating workloads with focus on specific wait_event_type? At least, say, CPU-bound, IO-bound workloads.

**Comment by lesovsky** [comment]
I think it depends on exact type of wait_event_type. If we talking about CPU- or IO-bound workloads (and resulting wait_events) it also could be tricky because directly depends on used hardware. It possible to generate the workload, but no strong guarantees that such workload could exhaust performance of particular (especially modern) server.
